bouncer/commands/go_bananas.ts

```diff
@@ -39,9 +40,14 @@ type AmountChange = null | {
   Increase?: number;
 };
 
+type ComplexAsset = {
+  chain: string;
+  asset: string;
+};
+
 type LimitOrderResponse = {
-  base_asset: string;
-  quote_asset: string;
+  base_asset: string | ComplexAsset;
+  quote_asset: string | ComplexAsset;
```

Comment:
why do we need two types here? is this to support legacy and new or?

Reply:
I can clean this up once I merge #4668 where I have done the heavy lifting to update all the assets.

Reply:
yes, the asset could be "USDC" or {chain: "Ethereum", asset: "USDT"}

Reply:
Yeah ok, as long we're tracking it and plan to remove it soon